Operations Documents/Effective Meetings.md

```diff
@@ -0,0 +1,46 @@
+# Effective meetings and group work
+
+To increase flexibility in how and where employees at Clef work, we agree on a shared set of constraints for group work and meetings which are strictly upheld by everyone on the team. 
+
+## Meeting Ettiquette
+
+### Meetings start on time
+
+If you're leading a meeting, it's your responsibility to start the meeting on time. If you're attending a meeting, you are responsible for showing up on time.
+
+### Meetings happen in regular hours
+
+All team members should make themselves available for face-to-face meetings between 10:30am-1:00p Pacific Time, as long as they are scheduled at least 24 hours in advance. If a meeting needs to happen outside of these hours, the meeting organizer should scheduled it a week in advance. 
+
+### Meetings should happen over video
+
+If you're hosting a meeting with a remote worker invited, it's your responsibility to provide a video link before the meeting starts for them to join. 
+
+If you're attending a meeting remotely, you should join the video call before a meeting starts. It's your responsibility to ensure this is possible — this means being in a quiet place with a fast internet connection before the meeting starts. Calling into a meeting via phone or from a public place is unacceptable.
+
+Morning stand ups are the exception to this and should happen over Slack calls. The meeting organizer can also make an exception when they create the meeting, but this should use this sparingly.
```

Comment:
I don't think we want the policy to be this specific -- I would just say something about voice calls being sufficient sometimes